src/test/java/io/vertx/core/http/HttpTestBase.java

```diff
@@ -34,7 +34,8 @@
 public class HttpTestBase extends VertxTestBase {
 
   public static final String DEFAULT_HTTP_HOST = "localhost";
-  public static final int DEFAULT_HTTP_PORT = 8080;
+  public static final int DEFAULT_HTTP_PORT = Integer.parseInt(System.getProperty("vertx.httpPort", "8080"));
+  public static final String DEFAULT_HTTP_HOST_AND_PORT = System.getProperty("vertx.httpHostPort", "localhost:8080");
```

Comment:
instead:

`public static final String DEFAULT_HTTP_HOST_AND_PORT = DEFAULT_HTTP_HOST + ":" +  DEFAULT_HTTP_PORT`

Reply:
and we should be using this wherever we can in tests